package.json

```diff
@@ -50,8 +49,7 @@
   },
   "husky": {
     "hooks": {
-      "pre-commit": "yarn run precommit",
```

Comment:
has opposite settings to our lint configurations that are checked in pre-push which caused me to be unable to push

Reply:
should we delete the task and dependencies too then?